scripts/update_feconf.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2017 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Helper script used for updating feconf.
+
+ONLY RELEASE COORDINATORS SHOULD USE THIS SCRIPT.
+
+Usage: Run this script from your oppia root folder:
+
+    python scripts/update_feconf.py
+"""
+
+import os
+
+import common # pylint: disable=relative-import
+
+
+FECONF_CONFIG_PATH = os.path.join(
+    os.getcwd(), os.pardir, 'release-scripts', 'feconf_updates.config')
+LOCAL_FECONF_PATH = os.path.join(os.getcwd(), 'feconf.py')
+
+
+def _apply_changes_to_feconf():
+    """Each line of the file should have a single '=' character and match a
+    line in feconf.py.
+    """
+    with open(FECONF_CONFIG_PATH, 'r') as f1:
+        config_lines = f1.read().strip().split('\n')
```

Comment:
Prefer to use ``f1.readlines()``. Ditto for ``f2``

Reply:
I started with that, but the lines all had \n at the end, and that made it a bit harder to debug things when printing to the terminal. I mildly prefer to keep it as it currently is... do you feel strongly?

Reply:
Alternative: https://stackoverflow.com/a/20756176/3689782

Might as well use ``splitlines()`` versus doing it manually.

Reply:
Good idea, I didn't know about splitlines()! Thanks, done.